core/linux-aarch64/PKGBUILD

```diff
@@ -33,7 +33,7 @@ md5sums=('d2985a3f16ef1ea3405c04c406e29dcc'
          '369fb1dd626c4e0a4a7e068d27dd51c8'
          '1553fe4abbe675ed8abd4644561d8e8c'
          '9844db3484f4d4fa3e63c7e083f656f7'
-         '5e0c36c663ebe0721fb96b9f2bfef451'
+         'SKIP'
```

Comment:
Please don't do this. This is the .config file, right?

Reply:
Yes, sorry about that. Fix the md5sum for `.config` and adjusted the pkgrel as well.